cli/src/main/java/org/jboss/as/cli/handlers/module/ASModuleHandler.java

```diff
@@ -707,4 +716,38 @@ public static void copy(final File source, final File target) throws CommandLine
         }
     }
 
-}
+    private String getSchemaVersion(CommandContext ctx, boolean hasSlot) {
+        if (detectedSchemaVersion == null) {
+            MXParser parser = new MXParser();
+            final File wfCommonModule = new File(modulesDir, "system.layers.base.org.wildfly.common.main.".replace('.', File.separatorChar) + "module.xml");
+            try {
+                parser.setFeature(FEATURE_PROCESS_NAMESPACES, true);
+                parser.setInput(new FileInputStream(wfCommonModule), null);
+                parser.nextTag();
+            } catch (Exception ignore) {
```

Comment:
You could list the set of exceptions and add a comment saying that it is safe to ignore the exception, we will simply not have a discovered schema.

Reply:
These kind of catches can lead to painful remote debugging sessions when end users run into issues, I think as @jfdenise says it would be better to restrict to the exceptions expected to be ignored as at the moment this will also catch all Runtime exceptions.

Regardless I think it would still be useful to log the exception at TRACE level then there remains a way to obtain information about what happened.